recipes/pyopenms/conda_build_config.yaml

```diff
@@ -0,0 +1,16 @@
+python:
```

Comment:
Just remove this file. I understand you might be excited for 3.11 but it is better to let the conda-forge tooling do it's work after the feedstock is accepted rather than trying to trick it to do it on the first merge.